public/about.html

```diff
@@ -2,17 +2,43 @@
 
 <html lang="en">
 
+<head>
+    <title>RERUM Geolocator</title>
+    <meta charset="UTF-8">
+    <link rel="icon" href="/images/favicon.ico" type="image/ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/chota@latest">
+    <link rel="stylesheet" href="stylesheets/style.css">
+    <script src="components/layout.js"></script>
+</head>
+
+<link rel="stylesheet" href="https://unpkg.com/chota@latest">
+<link rel="stylesheet" href="stylesheets/style.css">
+<header>
+    <div class="row" style="display: flex; justify-content: flex-start;">
+        <div>
+            <img src="https://centerfordigitalhumanities.github.io/rerum/logo.png" alt="logo">
+        </div>
+        <small class="rerum_shimmer">reconditorium eximium rerum universalium mutabiliumque</small>
+        <h1>
+            rerum geolocator
+        </h1>
+    </div>
+
+</header>
+
 <body is="about-page">
```

Comment:
You do not have a component called `about-page`.  What you want here is `<body is="geo-page">`